### PR TITLE
Added states to ar.xml according to ISO 3166-2.

### PR DESCRIPTION
**Pull request by rickygzz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please read below.
| Fixed ticket? | Fixes #14 (part of it, more PR's are coming :)

Added \<states> tag according to ISO 3166-2 (https://www.iso.org/obp/ui/#iso:code:3166:AR).

### CODE REVIEW

File: ar.xml [hunk 1]
```diff
@@ -29,6 +29,32 @@
       <taxRule iso_code_country="ar" id_tax="4"/>
     </taxRulesGroup>
 	</taxes>
+	<states>
+		<state name="Buenos Aires" iso_code="B" country="AR" zone="South America" />
+		<state name="Catamarca" iso_code="K" country="AR" zone="South America" />
+		<state name="Chaco" iso_code="H" country="AR" zone="South America" />
+		<state name="Chubut" iso_code="U" country="AR" zone="South America" />
+		<state name="Ciudad Autónoma de Buenos Aires" iso_code="C" country="AR" zone="South America" />
```
Comment (Julievrz) [inline ar.xml]: In the file https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/data/xml/state.xml it is called "Ciudad de Buenos Aires" only. @eternoendless should the name match too?

**Review by Julievrz — COMMENTED** [review]
All ISO codes in this file match the ones on https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/data/xml/state.xml. I only have a doubt regarding "Ciudad de Buenos Aires". I left a comment.

File: ar.xml (hunk 1 above)
Comment (rickygzz) [reply ar.xml]: State.xml file is used when installing PrestaShop for the first time.

It creates a few entries for countries' states inside \<prefix>_states database table. Not all countries are present.

In the other hand, the files in LocalizationFiles, are used when the user manually pulls data from the BackOffice.

You can see that, state.xml has all states for Mexico, but mx.xml localization has NO states at all (which will be fixed soon in other pull request). When you install prestashop, all MX states are mounted into the database.

(Also the iso codes for MX used in states.xml are outdated and will be updated soon in another PR)

Comment (eternoendless) [reply ar.xml]: The 2nd article of [the city's constitution](http://www.infoleg.gob.ar/?page_id=166) states that both names are correct: 

> ARTICULO 2º.- La Ciudad de Buenos Aires se denomina de este modo o como "Ciudad Autónoma de Buenos Aires".

While "Ciudad Autónoma de Buenos Aires" is the preferred official name, most people just use "Ciudad de Buenos Aires" for short. I'd go with the shorter name.

Comment (Julievrz) [reply ar.xml]: Ok then! 😇

Comment (rickygzz) [reply ar.xml]: I agree to go with the shorter name.

I made the changes. 

Happy holidays!

**Review by eternoendless — APPROVED** [review]
LGTM

**Review by Progi1984 — APPROVED** [review]

**Review by sLorenzini — APPROVED** [review]
hello @rickygzz,

Tested on develop and 178x branches
7.2.21 PHP version

- Add fix on ar.xml file in mystore folder > localization
- Go to locations > states > filter by country "Argentina"
- Deleted all states of Argentina
- Go to Localization > Select Argentina
- Choose "states" in content to import
- Set to "No" the "Download pack data" switch button (if set to yes not working, the states list of Argentina doesn't exist on prestashop.com)
- Click on Import button
- Go back to locations > states and update the page, all states on the list are displayed

QA approved ✅ 

Thank you

**Comment by Progi1984** [comment]
Thanks @rickygzz & @sLorenzini

**Pull request merged**